torch/quantization/fx/pattern_utils.py

```diff
@@ -37,6 +37,18 @@ def get_default_quant_patterns():
 def get_default_output_activation_post_process_map():
     return DEFAULT_OUTPUT_ACTIVATION_POST_PROCESS_MAP
 
+# a set of QuantizeHandler classes that are not observed
+# we'll skip inserting observers for input and output for these QuantizeHandlers
+# used for ops that only supports dynamic/weight only quantization
```

Comment:
We seem to be only  using this for weight-only quant right now. Do we plan on using this for dynamic quant as well?

Reply:
yeah this should be used for ops that **only** support dynamic quantization as well